### PR TITLE
[Zaraz] Zaraz Mmonitoring

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addresses PCX-4464

### CODE REVIEW

File: content/zaraz/monitoring.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+To use Zaraz Monitoring:
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login), and select your account and domain.
+2. Go to **Zaraz** > **Monitoring**.
+3. Select one of the options (Loads, Tracks, Triggers, Actions). Zaraz Monitoring will show you how the traffic for that section evolved for the time period selected.
+
+## Zaraz Monitoring options
+
+- **Loads**: Counts how many times Zaraz was loaded on pages of your website. When [Single Page Application support](/zaraz/reference/settings/#single-page-application-support) is enabled, Loads will count every change of navigation as well. 
+- **Tracks**: Counts how many times a specific event was tracked by Zaraz. It includes the [Pageview event](/zaraz/get-started/), [Track events](/zaraz/web-api/track/) and [E-commerce events](/zaraz/web-api/ecommerce/).
```
Comment (pedrosousa) [inline content/zaraz/monitoring.md]:
```suggestion
- **Tracks**: Counts how many times a specific event was tracked by Zaraz. It includes the [Pageview event](/zaraz/get-started/), [Track events](/zaraz/web-api/track/), and [E-commerce events](/zaraz/web-api/ecommerce/).
```
Nit: Not too relevant for this PR, but our internal glossary mentions that we use the "ecommerce" spelling.

File: content/zaraz/monitoring.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- **Triggers**: Counts how many times a specific trigger was activated. It includes the built-in [Pageview trigger](/zaraz/get-started/create-trigger/), and any other trigger you set in Zaraz.
+- **Actions**: Counts how many times a [specific action](/zaraz/get-started/create-actions/) was activated. It includes pre-configured Pageview actions, and any other action you set in Zaraz.
```
Comment (pedrosousa) [inline content/zaraz/monitoring.md]:
```suggestion
- **Actions**: Counts how many times a [specific action](/zaraz/get-started/create-actions/) was activated. It includes pre-configured Pageview actions and any other action you set in Zaraz.
```
Are there multiple "pre-configured Pageview actions", or just one?

File: content/zaraz/monitoring.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login), and select your account and domain.
+2. Go to **Zaraz** > **Monitoring**.
+3. Select one of the options (Loads, Tracks, Triggers, Actions). Zaraz Monitoring will show you how the traffic for that section evolved for the time period selected.
+
+## Zaraz Monitoring options
+
+- **Loads**: Counts how many times Zaraz was loaded on pages of your website. When [Single Page Application support](/zaraz/reference/settings/#single-page-application-support) is enabled, Loads will count every change of navigation as well. 
+- **Tracks**: Counts how many times a specific event was tracked by Zaraz. It includes the [Pageview event](/zaraz/get-started/), [Track events](/zaraz/web-api/track/) and [E-commerce events](/zaraz/web-api/ecommerce/).
+- **Triggers**: Counts how many times a specific trigger was activated. It includes the built-in [Pageview trigger](/zaraz/get-started/create-trigger/), and any other trigger you set in Zaraz.
```
Comment (pedrosousa) [inline content/zaraz/monitoring.md]:
```suggestion
- **Triggers**: Counts how many times a specific trigger was activated. It includes the built-in [Pageview trigger](/zaraz/get-started/create-trigger/) and any other trigger you set in Zaraz.
```

**Review by pedrosousa — APPROVED** [review]

File: content/zaraz/monitoring.md (hunk 1 above)
Comment (marciocloudflare) [reply content/zaraz/monitoring.md]: Yeah, I'm following what's in the dash :/

**Pull request merged**